app/routes.py

```diff
@@ -1,2 +1,184 @@
-from flask import Blueprint
+from werkzeug.wrappers import ETagRequestMixin
+from app.models.goal import Goal
+from flask import Blueprint, request, make_response, jsonify
+from app import db
+from app.models.task import Task
+from sqlalchemy import asc, desc
+from datetime import datetime
+import requests
+import os
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["GET","POST"])
+def handle_tasks():
+    if request.method == "GET":
+        sort = request.args.get("sort")
+
+        if sort == "asc":
+            tasks = Task.query.order_by(asc("title"))
+        
+        elif sort == "desc":
+            tasks = Task.query.order_by(desc("title"))
+        
+        else:
+            tasks = Task.query.all()
+
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": task.is_complete()
```

Comment:
We could use the task instance method here `task.make_json` to DRY up the code and reduce room for errors.